addons/binding/org.openhab.binding.regoheatpump/README.md

```diff
@@ -0,0 +1,222 @@
+# RegoHeatPump Binding
+
+The Rego heat pump binding supports:
+* Rego 6xx controllers family and
+* Husdata interface.
+
+***Disclaimer*** - *there is no guarantee for the function of the binding or its correctness. Use at your own risk.*
+
+## The Rego 6xx family
+
+The Rego 6xx controllers family is used in many heat pumps, like IVT/Bosch/Autotherm/Carrier and others.
+
+Rego 6xx unit contain an interface marked as service. Header of this interface is close to the control unit. This is 5V (TTL) serial interface and is connected by a 9 pin can/d-sub connector. Pinout:  
+
+2 - RxD  
+3 - TxD  
+4 - +5V  
+5 - GND
+
+Serial communication is using 19200 bps, 8 bit, no parity, 1 stop bit.
+
+### Thing configuration
+
+Two connection types are supported:
+* TCP/IP and
+* serial (RS232).
+
+#### TCP/IP connection
+
+A transparent bridge between the serial interface of the heat pump and network (i.e. wifi) is used. This way no additional wires are required between heat pump and computer, running openhab.
+
+There are many existing project providing such functionality, i.e. [ser2net](http://ser2net.sourceforge.net/).
+
+For my setup, I used a low budget (~5€) circuit, that is integrated into the heat pump and connects to a wifi using an ESP8266 based module.
+
+Board:
+
+![board](doc/board.png)
+
+The code running on the ESP module can be found [here](https://github.com/crnjan/esp8266-bridge). There are other projects providing ESP firmware with similar functionality, i.e. [ESP-LINK](https://github.com/jeelabs/esp-link), but did not test with those.
+
+Configuration of the TCP/IP thing:
+
+ - address: the hostname/IP address of the transparent bridge on the local network - mandatory,
+ - tcpPort: the port number to use to connect to the transparent bridge - optional, defaults to 9265,
+ - refreshInterval: refresh interval in seconds, used to fetch new values from the heat pump - optional, defaults to 60 seconds.
+
+Example thing definition:
+ ```
+ regoheatpump:ipRego6xx:ivtIP [ address="192.168.2.50", tcpPort="9265" ]
+ ```
+
+#### Serial connection
+
+In order to connect directly to the rego 6xx controller, one needs to adjust the TTL levels coming from the rego unit to levels used by a RS232 serial port, used within computers, using MAX232 or similar.
+
+Parameters:
+
+ - portName: the name of the serial port on your computer - mandatory,
+ - refreshInterval: refresh interval in seconds, used to fetch new values from the heat pump - optional, defaults to 60 seconds.
+
+Example thing definition:
+
+```
+regoheatpump:serialRego6xx:ivtSerial [ portName="COM3" ]
+```
+
+
+### Channels
+
+Below is the list of supported channels, all values are read only:
+
+| Channel Type ID        | Item Type           
```

Comment:
You would make me happy by formatting the tables in the MD file as well :-D

http://www.tablesgenerator.com/markdown_tables

Select 'File' > 'Past table data ...' and past the unformatted mark down table, you can then copy paste the formatted table 👍